### DISCUSSION TITLE
proxy_pass to `http://remark42:8080/` forced to `https` by Chrome

### DISCUSSION THREAD

**Discussion by camachom** [umputun/remark42#1670] [Q&A] [answered]
Hello! Thanks for maintaining this awesome project. 

My blog is running nginx and remark through docker compose per the docs. The only unusual thing is that my blog is being served from another subdomain. So remark runs on `remark42.vs.computer` and my Hugo site runs on `me.vs.computer`. I had to allow CORS which feels unusual. 

nginx config: https://github.com/camachom/camachom.github.io/blob/master/nginx.conf
docker-compose: https://github.com/camachom/camachom.github.io/blob/master/docker-compose.yml

In Chrome, its forcing `http://remark42:8080/` into `https://remark42:8080/` which obviously breaks:
<img width="1440" alt="Screenshot 2023-09-01 at 10 31 47 AM" src="https://github.com/umputun/remark42/assets/15637784/b8f9cb7b-5e2d-4294-9d3e-ea25e38455e5">

In Safari, its able to request `http://remark42:8080/`, but its not found:
<img width="1440" alt="Screenshot 2023-09-01 at 10 33 08 AM" src="https://github.com/umputun/remark42/assets/15637784/e2fc814c-f0ba-4ed9-96ee-eb91224a8169">

What am I doing wrong? I really appreciate the help

**Comment by umputun** [comment, marked as answer]
My guess would be `REMARK_URL=http://remark42:8080` - this seems to reference the internal URL, but you need the external one here, i.e. smth like "https://remark42.vs.computer"

**Reply by camachom** [reply]
So I see that remark is picking up the change:
```
remark42  | 2023/09/01 13:01:30.430 [INFO]  {cmd/server.go:309 cmd.(*ServerCommand).Execute} start server on port :8080
remark42  | 2023/09/01 13:01:30.431 [INFO]  {cmd/server.go:480 cmd.(*ServerCommand).newServerApp} root url=https://remark42.vs.computer
```

but I still get the exact same issue. Still tries to make requests for `http://remark42:8080/api/v1/avatar/6ec042bc9a764527877624aa4569ddb40acc5498.image` that fail

**Reply by umputun** [reply]
I guess this is for the comment you created before the change? if this is the case you can clean the db and try it again

**Reply by camachom** [reply]
thank you so much!
